desc/perturbations.py

```diff
@@ -427,6 +427,7 @@ def optimal_perturb(  # noqa: C901 - FIXME: break this up into simpler pieces
     dPsi=False,
     dRb=False,
     dZb=False,
+    dIGPhi=False,
```

Comment:
not in docstring, and is not included in the `argmap`, is this necessary/used in this PR?

Reply:
no this is leftover from an earlier version, will remove.